agent/util-scripts/pbench-tool-meister-start

```diff
@@ -773,6 +790,21 @@ def main(_prog, cli_params):
     except KeyError as exc:
         logger.error("failed to fetch required environment variable, '%s'", exc.args[0])
         return ReturnCode.MISSINGREQENVS
+    if not full_hostname or not hostname:
+        logger.error(
+            "_pbench_hostname ('%s') and _pbench_full_hostname ('%s')"
+            " environment variables are required to represent the respective"
+            " hostname strings",
+            hostname,
+            full_hostname,
+        )
+        return ReturnCode.MISSINGHOSTNAMEENVS
```

Comment:
Not sure it is worth combining all this. We want the user to know that we expect both `_pbench_hostname` and `_pbench_full_hostname` defined.  They could certainly infer that from encountering both error messages by executing things twice, but this log message explains the requirement.  The validation individually explains the requirement that valid host names be used.

Reply:
So, `logger.error()` is going to send this to `stderr` or somewhere else where the end-user will see it?  (I keep falling under the misconception that "the log" gets squirreled away where we can ask for it, but it's otherwise mostly out of sight; I was trying to save code because we already know most of what this message is trying to say -- we just want the `%s` values, which I think we would get from the code later on.)

From what you say here, it seems like it would be worthwhile to enhance this:  report on each individually -- but report them both if they are both bad -- and go ahead and reference the `hostname -s` and `hostname -f` commands specifically, like you did elsewhere.

Reply:
See line 763 where we use the `StreamHandler` for the logger.